12LessonTwelve/homework/index.html

```diff
@@ -1,10 +1,77 @@
+
+    <!-- Google font-->
+    <link href="https://fonts.googleapis.com/css?family=Lily+Script+One" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <div class="row">
+        <div class="col-sm-12">
+            <header>
+                <h1>Best Selling Products</h1>
+            </header>
+        </div>
+    </div>
+    <section id="sales">
+        <div class="container">
+            <div class="row">
+                <div class="box col-lg-3 col-sm-6">
+                    <div class="picture">
+                        <div class="price">$35.00</div>
+                        <img src="http://inspirythemes.biz/html-templates/cream/images/temp-images/hoodie_2_front-300x300.jpg" alt="Hoodie" />
+                        <h4>Happy Ninja</h4>
+                    </div>
+                    <div class="shop">
+                        <a href="#">ADD TO CART</a>
+                    </div>
+                </div>
+                <div class="box col-lg-3 col-sm-6">
+                    <div class="picture">
+                        <div class="second price">$35.00</div>
+                        <img src="http://inspirythemes.biz/html-templates/cream/images/temp-images/hoodie_3_front-300x300.jpg" alt="Hoodie" />
+                        <h4>Patient Ninja</h4>
+                    </div>
+                    <div class="shop">
+                        <a href="#">ADD TO CART</a>
+                    </div>
+                </div>
+                <div class="box col-lg-3 col-sm-6">
+                    <div class="picture">
+                        <div class="sale">SALE!</div>
+                        <div class="price">$15.00</div>
+                        <img src="http://inspirythemes.biz/html-templates/cream/images/temp-images/poster_1_up-300x300.jpg" alt="Hoodie" />
+                        <h4>Ship Your Idea</h4>
+                    </div>
+                    <div class="shop">
+                        <a href="#">ADD TO CART</a>
+                    </div>
+                </div>
+                <div class="box bottom col-lg-3 col-sm-6">
+                    <div class="picture">
+                        <div class="last price"><span>$15.00</span>$12.00</div>
+                        <img src="http://inspirythemes.biz/html-templates/cream/images/temp-images/poster_2_up-300x300.jpg" alt="Hoodie" />
+                        <h4>Flying Ninja</h4>
+                    </div>
+                    <div class="shop">
+                        <a href="#">ADD TO CART</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 
 </body>
+
 </html>
```

Comment:
Looks good. Just one small change to the pricing box when the media queries change.
<img width="829" alt="screen shot 2016-09-13 at 5 16 16 pm" src="https://cloud.githubusercontent.com/assets/12698279/18493382/f72d0dc8-79d5-11e6-9491-0f8bbb37b217.png">
<img width="338" alt="screen shot 2016-09-13 at 5 16 26 pm" src="https://cloud.githubusercontent.com/assets/12698279/18493383/f72d312c-79d5-11e6-80a6-e30b28ce11d9.png">